src/rpc/mining.cpp

```diff
@@ -1017,6 +1017,7 @@ UniValue getnewblockhex(const JSONRPCRequest& request)
                             },
                         },
                         "proposed_parameters"},
+                        {"data", RPCArg::Type::STR_HEX, RPCArg::Optional::NO, "Data in hex to be committed to in an additional coinbase output."},
```

Comment:
nit: `commit_data`

Reply:
that's not a nit :) will fix

Reply:
oh I see, you're nitting the name. I'm merely matching what's in `createrawtransaction` fwiw, so I'll keep this if that's ok

Reply:
I was nitting the name. In createrawtx, the `data` field is part of an output. Here it's part of a block. I think it's less obvious what "data" means in a "create block" context. But well if you think consistency is more important, it's fine. ack then.

Reply:
ok, changed